.ibm/pipelines/install-methods/operator.sh

```diff
@@ -18,17 +19,38 @@ install_rhdh_operator() {
   chmod +x /tmp/install-rhdh-catalog-source.sh
   if [[ "$RELEASE_BRANCH_NAME" == "main" ]]; then
     echo "Installing RHDH operator with '--next' flag"
-    bash -x /tmp/install-rhdh-catalog-source.sh --next --install-operator rhdh
+    for ((i = 1; i <= max_attempts; i++)); do
```

Comment:
> Consider adding a delay between retry attempts to avoid overwhelming the system:
> ```bash
> elif ((i < max_attempts)); then
> echo "Attempt ${i} failed, retrying in 10 seconds..."
> sleep 10
> fi
> ```

Reply:
In my testing, this wasn't making any difference, but 10 seconds won't hurt, so I'm adding it.